hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -213,50 +167,70 @@ void fillBuffer(TimestampedSourceBuffer<T> buf) throws Exception {
         }
     }
 
-    private static BatchSource<Long> longBatchSource(long range, int stepSize) {
-        return SourceBuilder
-                .batch("longs", c -> new LongAccumulator())
-                .<Long>fillBufferFn(new LongSource(range, stepSize)::fillBufferFn)
-                .build();
+    /**
+     * Returns a stream source that contains long values and does late materialization
+     * of values after distributing them across the cluster, which is useful for high-throughput
+     * testing.
+     *
+     * @since 4.3
+     *
+     */
+    @Nonnull
+    public static StreamSource<Long> longStreamSource(long itemsPerSecond, long initialDelay) {
+        return longStreamSource(itemsPerSecond, initialDelay, Vertex.LOCAL_PARALLELISM_USE_DEFAULT, false);
     }
 
-    private static StreamSource<Long> longStreamSource(long range, int stepSize) {
-        return SourceBuilder
-                .stream("longs", c -> new LongAccumulator())
-                .<Long>fillBufferFn(new LongSource(range, stepSize, true)::fillBufferFn)
-                .build();
+    /**
+     * Returns a stream source that contains long values and does late materialization
+     * of values after distributing them across the cluster, which is useful for high-throughput
+     * testing.
+     *
+     * @since 4.3
+     *
+     */
+    @Nonnull
+    public static StreamSource<Long> longStreamSource(long itemsPerSecond, long initialDelay,
```

Comment:
Wrap the params like this:

```java
public static StreamSource<Long> longStreamSource(
    long itemsPerSecond, long initialDelay, int preferredLocalParallelism
) {
```

Reply:
Done w/ 8fcf10859c9274eca0f80fe5f7b45bac375000dc